conf/broker.conf

```diff
@@ -188,7 +188,7 @@ brokerDeleteInactiveTopicsMode=delete_when_no_subscriptions
 brokerDeleteInactivePartitionedTopicMetadataEnabled=false
 
 # Max duration of topic inactivity in seconds, default is not present
-# If not present, 'brokerDeleteInactiveTopicsFrequencySeconds' will be used
```

Comment:
Should not change it

Reply:
https://github.com/apache/pulsar/blob/f230d15ffcd5f74cca13bd23b35ace784d6f8ce6/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java#L2834-L2840